draft-ietf-tls-tls13.md

```diff
@@ -2001,42 +2032,36 @@ interoperability failure.
 
 ###  Hello Retry Request
 
-The server will send this message in response to a ClientHello message if it is
-able to find an acceptable set of parameters but the ClientHello does not
-contain sufficient information to proceed with the handshake.
-
-Structure of this message:
-
-%%% Key Exchange Messages
-
-       struct {
-           ProtocolVersion server_version;
-           CipherSuite cipher_suite;
-           Extension extensions<2..2^16-1>;
```

Comment:
One detail I noticed as I was sorting out some of our tests: at one point we had text which disallowed no-op HelloRetryRequests. When HRR got moved to extensions, this turned into just a rule that the extensions block is non-empty. (That plus no unsolicited extensions did the trick.)

This is gone now, but it actually wouldn't have sufficed anyway because every HRR has a supported_versions extension now. So we probably need to go back to explicit text here somewhere.

Reply:
This is on line 2053.